composer.json

```diff
@@ -16,6 +16,7 @@
         "ext-libxml": "*",
         "ext-mbstring": "*",
         "ext-openssl": "*",
+        "demos-europe/demosplan-addon": "dev-upgrade-edt-to-0.25",
```

Comment:
needs to be changed before merging

Reply:
this is fixed too, and all unnecessary comments are removed